api/v1alpha1/hardwareclassification_types.go

```diff
@@ -115,12 +115,12 @@ type Ram struct {
 	// +kubebuilder:validation:Minimum=1
 	// MinimumSizeGB of Ram should be greater than 0
 	// Ex. MinimumSizeGB > 0
-	MinimumSizeGB int `json:"minimumSizeGB,omitempty"`
+	MinimumSizeGB int64 `json:"minimumSizeGB,omitempty"`
```

Comment:
I suggest to defind int32 instead of int64

Reply:
Done